src/app/pages/details/common/resource-box/resource-box-utils.js

```diff
@@ -1,31 +1,11 @@
 import React from 'react';
 import {FormattedMessage} from 'react-intl';
 import linkHelper from '~/helpers/link';
-import RawHTML from '~/components/jsx-helpers/raw-html';
 import {useDataFromSlug, camelCaseKeys} from '~/helpers/page-data-utils';
-import LeftContent from './left-content';
 import useUserContext from '~/contexts/user';
-import {FontAwesomeIcon} from '@fortawesome/react-fontawesome';
-import {faShoppingCart} from '@fortawesome/free-solid-svg-icons/faShoppingCart';
-import cn from 'classnames';
 
 const settings = window.SETTINGS;
 
-export default function ResourceBox({model}) {
```

Comment:
I split the components out into resource-box.tsx, and left the utilities in here, renaming it resource-box-utils.js It will go to TS in the next PR.